opendebates/celery.py

```diff
@@ -0,0 +1,26 @@
+"""Initialize celery"""
+from __future__ import absolute_import
+
+import os
+
+from celery import Celery
+
```

Comment:
Should we add `dotenv` here, too? We have it manage.py. If so, we should include it in wsgi.py as well.